docs/vscode.md

```diff
@@ -230,6 +230,27 @@ if (aligner == 'bowtie2') {
 }
 ```
 
+**Spread operator**
+
+Groovy supports a "spread" operator, which can be used to flatten a list in certain situations:
```

Comment:
```suggestion
In Groovy, the "spread" operator can be used to flatten a list:
```

Reply:
Does this only apply in certain situations? Because the certain situations aren't addressed explicitly I've removed it, but if you think it's an important point we can add it back/